### ISSUE TITLE
Remove twisted.words.protocols.msn

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8253]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#8253|
|Type|task|
|Created|2016-03-25 15:07:31Z|
|Branch|https://github.com/twisted/twisted/tree/rip-in-pepperoni-msn-8253|

It's dead, and dead for good. Let's drop it.

<details><summary>Searchable metadata</summary>

```
trac-id__8253 8253
type__task task
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_rip_in_pepperoni_msn_8253 branches/rip-in-pepperoni-msn-8253
branch_author__hawkowl hawkowl
status__closed closed
resolution__fixed fixed
component__words words
keywords__None None
time__1458918451564439 1458918451564439
changetime__1459225874603138 1459225874603138
version__None None
owner__hawkowl hawkowl
cc__ralphm
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

(In [47074]) Branching to rip-in-pepperoni-msn-8253.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

Yeah, I'm happy with this. Let's put it to bed.

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

Note for reviewers: this branch introduces no new twistedchecker errors, the builder is just freaking out because so many twistedchecker errors were removed

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set owner to @hawkowl|
|-|-|

LGTM. Please merge.

Many thanks for the cleanup!

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_6"></a>|@hawkowl set status to `closed`|
|-|-|

(In [47099]) Merge rip-in-pepperoni-msn-8253: Remove MSN (twisted.words.protocols.msn)

Author: hawkowl
Reviewer: adiroiban
Fixes: [#8253](https://github.com/twisted/twisted/issues/8253)
